integration-tests/deploy.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python3
+import os, argparse, sys, subprocess, shlex, docker,  requests, json
+from python_on_whales import docker as d_cli
+from docker import errors as d_err
+from docker import types as d_types
+from retry import *
+
+
+def request_metrics(port):
+    try:
+        return requests.get(f"http://0.0.0.0:{port}/metrics")
```

Comment:
Again, I expect `0.0.0.0` should not work here. According to my expectation it should be `127.0.0.1`. Why does it work?

Reply:
fixed